R/mod_popExp_heatCorr.R

```diff
@@ -110,6 +110,13 @@ heatmap_srv <- function(input, output, session, data, run) {
     }) %>%
       na.omit() %>% as.character()
```

Comment:
I have come across this issue several times where `toJSON()` will not translate a vector of length one to an array

```
# This 
shiny:::toJSON(list(`Time Dependent` = c("HEIGHT", "WEIGHT")))
# {"Time Dependent":["HEIGHT","WEIGHT"]}
# versus this
shiny:::toJSON(list(`Time Dependent` = "HEIGHT"))
# {"Time Dependent":"HEIGHT"}
```
I personally have found passing lists instead of vectors to be the simpler solution.
```
shiny:::toJSON(list(`Time Dependent` = as.list("HEIGHT")))
# {"Time Dependent":["HEIGHT"]} 
```

In other words we can pipe `paramcd` to a list (i.e. line 111 as `na.omit() %>% as.character() %>% as.list()`) and correct the issue without making it a named vector.

Reply:
@Jeff-Thompson12  Thanks for the tip. I have changed `paramcd` to a list and that works.  Ready for review again once the GHAs finish.